### PR TITLE
给 deconvolution 一个介绍。

### PR DESCRIPTION
**Pull request by heroxbd**
Closes: https://github.com/heroxbd/waveform-analysis/issues/107

### CODE REVIEW

File: Note/Algorithm.tex [hunk 1]
```diff
@@ -107,11 +107,11 @@ \subsubsection{Peak finding}
...
+Deconvolution is motivated by viewing the waveform as a convolution of sparse spike train $\tilde{\phi}$ and $V_\mathrm{PE}$ in eq.~\eqref{eq:1}.  Huang et al.~\cite{huang_flash_2018} from DayaBay and Grassi et al.~\cite{grassi_charge_2018} introduced deconvolution-based waveform analysis in charge reconstruction and linearity studies.  Zhang et al.~\cite{zhang_comparison_2019} then applied it to JUNO prototype.  Deconvolution methods are better than heuristic ones by using the full shape of $V_\mathrm{PE}(t)$, thus can accommodate overshoots and pile-ups.  Noise and Nyquist limit make deconvolution sensitive to fluctuations in real-world applications.  A very careful selection of low-pass filter mitigates the difficulty but might introduce Gibbs ringing artifects in the smoothed waveforms and the deconvoluted results. Despite such draw-backs, deconvolution algorithms are fast and useful to give initial crude solutions for the more advanced algorithms.  Deployed in running experiments, they are discussed in this section to make an objective evaluation.
```
[the remark this answers is not in the document]
Comment (dachengx) [reply Note/Algorithm.tex:110]: filter -> filters?

[the remark this answers is not in the document]
Comment (dachengx) [reply Note/Algorithm.tex:110]: s/artifects/artifacts

[the remark this answers is not in the document]
Comment (dachengx) [reply Note/Algorithm.tex:110]: draw-backs -> drawbacks?

File: Note/Algorithm.tex [hunk 2]
```diff
@@ -75,14 +75,14 @@ \subsubsection{Wasserstein distance}
 
 \input{fom}
 
-In the following, we  evaluate the performance of waveform analysis algorithms ranging from heuristics, deconvolution, neural network to regression by the criteria discussed in this section.
```
Comment (greatofdream) [inline Note/Algorithm.tex:78]: evaluate performace换成评估？In the following part the performance of waveform analysis algorithms, including ..., are estimated/assessed using the criteria discussed in this section.

File: Note/Algorithm.tex (hunk 1 above)
Comment (greatofdream) [inline Note/Algorithm.tex:110]: Deconvolution is motivated by the perspective of viewing the waveform as a convolution .... with white noise.

[the remark this answers is not in the document]
Comment (greatofdream) [reply Note/Algorithm.tex:110]: > the JUNO prototype

这里是我之前写的不严谨，我看论文全称是  the JUNO prototype detector

[the remark this answers is not in the document]
Comment (greatofdream) [reply Note/Algorithm.tex:110]: > filter -> filters?

我猜这里续老师想表达一个filter，但是这里A修饰到了selection

[the remark this answers is not in the document]
Comment (littlewu2508) [reply Note/Algorithm.tex:110]: a selection of low pass filters mitigate ...

or

Carefully chosen a low pass filter mitigates ..

**Comment by heroxbd** [comment]
littlewu2508 ***@***.***> writes:

> 我没有其它建议了

收到，我晚上把它集成上去。

File: Note/Algorithm.tex (hunk 1 above)
[the remark this answers is not in the document]
Comment (heroxbd) [reply Note/Algorithm.tex:110]: 改成了 A carefully selected low-pass filter

File: Note/Algorithm.tex (hunk 2 above)
Comment (heroxbd) [reply Note/Algorithm.tex:78]: we assess the performance of ...

File: Note/Algorithm.tex (hunk 1 above)
Comment (heroxbd) [reply Note/Algorithm.tex:110]: 感觉语气没问题，我先保留这句话了。

**Pull request closed without merging**